themes/bootstrap3/js/combined-search.js

```diff
@@ -5,6 +5,11 @@ VuFind.combinedSearch = (function CombinedSearch() {
     VuFind.loadHtml(container, url, '', function containerLoad(responseText) {
       if (responseText.length === 0) {
         container.hide();
+        container = container.parent();
+        while (container.hasClass('hide-if-empty')) {
+          container.hide();
+          container = container.parent();
+        }
```

Comment:
I know we are moving to eliminate JQuery from the codebase.  I used it here because this function is called with a JQuery element, and VuFind.initResultScripts below also expects a JQuery element, so with input and output determined it seemed a bad place to fix.